src/lib/mina_graphql/mina_graphql.ml

```diff
@@ -3646,11 +3658,12 @@ module Mutations = struct
       ~doc:"Send a zkApp (for internal testing purposes)"
       ~args:
         Arg.
-          [ arg "parties" ~typ:(non_null Types.Input.SendTestZkappInput.arg_typ)
+          [ arg "zkapp_command"
```

Comment:
this one is a bit less important (so if it complicates other code, definitely leave it as is), but probably also should be `zkappCommand`